src/partials/gallery.html

```diff
@@ -1 +1,43 @@
-<section class="gallery">gallery</section>
\ No newline at end of file
+<section class="gallery">
+  <img
+    src="images/gallery/gallery-tablet@2.png"
+    alt="gallery"
+    class="gallery__img"
+    srcset="
+      images/gallery/gallery-mobile.png     320w,
+      images/gallery/gallery-tablet.png     768w,
+      images/gallery/gallery-desktop.png   1280w,
+
+      images/gallery/gallery-mobile@2.png   640w,
+      images/gallery/gallery-tablet@2.png  1536w,
+      images/gallery/gallery-desktop@2.png 2560w
+    "
+    sizes="
+    (max-width:   767px) 320px,
+    (min-width: 768px) and (max-width: 1199px) 768px,
```

Comment:
max-width tu chyba powinien być 1279px, jak w linii poniżej, bo teraz nasz punkt przegięcia to 1280px?